src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/APIEndpoint.kt

```diff
@@ -53,7 +53,7 @@ data class APIEndpoint(
 
         val config = factory.build()
 
-        spark.before(url, org.pac4j.sparkjava.SecurityFilter(
+        spark.before(url, contentType, org.pac4j.sparkjava.SecurityFilter(
```

Comment:
One of the new endpoints POST `/report/:name/tag` differs from the report page url GET `report/:name/:version` in verb and content type (json vs html) but not in route params - discovered this bug where the security rule was not distinguising such routes (although the router itself does). Meaning the security rule for the latter was being applied to the new endpoint. This fixes that by adding content type into the security rule. I do feel like it should really also distinguish by verb but it looks like Spark doesn't support that.

Reply:
Really? That's bad. Surely that sort of thing is what REST is based on. Does that mean it won't be able to distinguish json POST and GET of report/:name/version/:version/tags where both are json but only the GET should be permitted to readers? (Now that you've updated /tag to /tags...)

Reply:
Correct, I can't see see a way to add a Spark route filter based on verb

Reply:
Oh yeah, that will be a problem, ugh